sql_generators/active_users/templates/metadata.yaml

```diff
@@ -30,7 +30,7 @@ bigquery:
     require_partition_filter: true
   clustering:
     fields:
-    - country
-    - app_name
-    - attribution_medium
-    - channel
+      - country
```

Comment:
@bochocki, a small update in the formatting inside /active_users that requires your approval.